ting_search.module

```diff
@@ -973,5 +973,15 @@ function ting_search_submit($form, &$form_state) {
  */
 function _ting_search_quote($string) {
   $cqlDoc = new TingSearchCqlDoctor($string);
-  return $cqlDoc->string_to_cql($string);
+  // Hold cql query for further changes.
+  $return = $cqlDoc->string_to_cql();
```

Comment:
I am wondering whether this method is the right place to implement the change. What are your thoughts about this?

My considerations:

- This function `_ting_search_quote()` has to do with quoting of the query according to the name and cql strictness according to the DocBlock. The change is not related to any of those as I see it.
- The issue states that the `holdingsitem` filter should be added to *all* queries sent to OpenSearch. When added to this method it is only ensured that it is added to user submitted queries but OpenSearch is queried in other situations as well. I wonder if it would be more appropriate to add it to [`ting_do_search()`](https://github.com/ding2/ting/blob/master/ting.client.inc#L227) in the general `ting` module. I notice that the variable has also been added there.